dev/com.ibm.ws.security.authorization.jacc.ejb/test/com/ibm/ws/security/authorization/jacc/ejb/impl/DummyPolicyConfigurationFactory.java

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
+ * Copyright (c) 2015 IBM Corporation and others.
```

Comment:
Do the copyrights of the test files need to be updated to have ,2020 on them as well?

Reply:
Yes. Done.